### DISCUSSION TITLE
Custom profile error when loading:  Expecting property name enclosed in double quotes: "default_black_out_nsfw": true,

### DISCUSSION THREAD

**Discussion by DisasterofPuppets** [lllyasviel/Fooocus#3677] [Q&A] [answered]
I keep getting an error in the console when trying to run a custom .json profile.
Error and .json below

Any help is greatly appreciated. Thank you.

This is the error I get:

[System ARGV] ['Fooocus\\entry_with_update.py', '--preset', 'default']
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Fooocus version: 2.5.5
Load default preset failed.
Expecting property name enclosed in double quotes: line 61 column 1 (char 1767)
Load preset [Y:\Fooocus_win64_2-5-0\Fooocus\presets\default.json] failed
Expecting property name enclosed in double quotes: line 61 column 1 (char 1767)

This last line (before the }) is the part in question (removing it allows the script to run without the error)

{
    "default_model": "juggernautXL_v8Rundiffusion.safetensors",
    "default_refiner": "None",
    "default_refiner_switch": 0.5,
    "default_loras": [
        [
            true,
            "sd_xl_offset_example-lora_1.0.safetensors",
            0.1
        ],
        [
            true,
            "None",
            1.0
        ],
        [
            true,
            "None",
            1.0
        ],
        [
            true,
            "None",
            1.0
        ],
        [
            true,
            "None",
            1.0
        ]
    ],
    "default_cfg_scale": 4.0,
    "default_sample_sharpness": 2.0,
    "default_sampler": "dpmpp_2m_sde_gpu",
    "default_scheduler": "karras",
    "default_performance": "Speed",
    "default_prompt": "",
    "default_prompt_negative": "",
    "default_styles": [
        "Fooocus V2",
        "Fooocus Enhance",
        "Fooocus Sharp"
    ],
    "default_aspect_ratio": "1152*896",
    "default_overwrite_step": -1,
    "checkpoint_downloads": {
        "juggernautXL_v8Rundiffusion.safetensors": "https://huggingface.co/lllyasviel/fav_models/resolve/main/fav/juggernautXL_v8Rundiffusion.safetensors"
    },
    "embeddings_downloads": {},
    "lora_downloads": {
        "sd_xl_offset_example-lora_1.0.safetensors": "https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0/resolve/main/sd_xl_offset_example-lora_1.0.safetensors"
    },
    "previous_default_models": [
        "juggernautXL_version8Rundiffusion.safetensors",
        "juggernautXL_version7Rundiffusion.safetensors",
        "juggernautXL_v7Rundiffusion.safetensors",
        "juggernautXL_version6Rundiffusion.safetensors",
        "juggernautXL_v6Rundiffusion.safetensors"
    ],
	"default_black_out_nsfw": true,
}

**Comment by Rogala** [comment, marked as answer]
`"default_black_out_nsfw": true` 
Remove the comma at the end

**Reply by DisasterofPuppets** [reply]
Champ, that fixed it.
Thank you.
